tests/rules/no-important.test.js

```diff
@@ -82,6 +94,12 @@ ruleTester.run("no-important", rule, {
 					column: 34,
 					endLine: 1,
 					endColumn: 45,
+					suggestions: [
+						{
+							messageId: "removeImportant",
+							output: "a { padding: 10px 20px 30px 40px ; }",
```

Comment:
I think when we remove `!important` we should also remove any whitespace before the `!` to avoid these gaps.